airflow/providers/amazon/aws/operators/glacier.py

```diff
@@ -54,3 +54,55 @@ def __init__(
     def execute(self, context: Context):
         hook = GlacierHook(aws_conn_id=self.aws_conn_id)
         return hook.retrieve_inventory(vault_name=self.vault_name)
+
+
+class GlacierUploadArchiveOperator(BaseOperator):
+    """
+    This operation adds an archive to a vault
+
+    .. seealso::
+        For more information on how to use this operator, take a look at the guide:
+        :ref:`howto/operator:GlacierUploadArchiveOperator`
+
+    :param account_id: The AWS account ID of the account that owns the vault.
+        You can either specify an AWS account ID or optionally a single `-` (hyphen),
+        in which case Amazon S3 Glacier uses the AWS account ID associated with
+        the credentials used to sign the request
```

Comment:
```suggestion
    :param account_id: Optional AWS account ID of the account that owns the vault.  Defaults to the credentials used to sign the request
```

Reply:
Is there a reason the hyphen was used here, or is it arbitrary?   If it's arbitrary, it should be changed to an optional string and allow None instead.

Reply:
anyway AWS API add `-` by default so we can make it optional